TelCo.ColorCoder/ManualMap.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+  class ManualMap{
```

Comment:
It is good you have separated Manual from the main class.
But name ManualMap indicates its a data structure, but the class does not hold any data.
Can change the name of the class.